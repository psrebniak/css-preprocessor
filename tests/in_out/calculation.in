$v1: 640;
$v2: 2;

.col-1-4 {
    width: (${v1} / 4);
}
.col-2-4 {
    width: (2 * (${v1} / 4));
}
.col-3-4 {
    width: (3 * (${v1} / 4));
}
.col-4-4 {
    width: (4 * (${v1} / 4));
}

.text_test {
    test: (test + (4 * (${v1} / 4)));
}
