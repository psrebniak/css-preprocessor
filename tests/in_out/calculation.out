.col-1-4 {
    width: 160;
}
.col-2-4 {
    width: 320;
}
.col-3-4 {
    width: 480;
}
.col-4-4 {
    width: 640;
}
.text_test {
    test: test640;
}
